pandas/core/arrays/base.py

```diff
@@ -236,6 +236,18 @@ def isna(self):
         """
         raise AbstractMethodError(self)
 
+    def unique(self):
+        """Compute the ExtensionArray of unique values.
+
+        Returns
+        -------
```

Comment:
future PR should prob add some examples here :> (and other doc-strings).

Reply:
True :) The only problem is that for ExtensionArray we don't have a direct working example, as you first need to subclass it (unless we use one of the existing ones like Categorical, but that also seems a bit strange)